tests/integration/cluster_fixture.py

```diff
@@ -273,7 +286,7 @@ def seed_topic(self, topic, value_source=None, key_source=None, header_source=No
             value_source = ['test-data{}'.format(i) for i in range(0, 100)]
 
         if key_source is None:
-            key_source = [None]
+            key_source = ['test-key{}'.format(i) for i in range(0, 100)]
```

Comment:
Let's avoid changing this function that is used in other places and pass a `key_source` and `value_source` this time, or otherwise if you change it let's change the other invocations so you don't add keys to those tests.